### DISCUSSION TITLE
The RA and Dec of the Moon have nothing to do with the observation position, right?

### DISCUSSION THREAD

**Discussion by DieForSpicy** [skyfielders/python-skyfield#686] [Q&A]
Hello! I'm new in astronomy and there is a problem bothering me.
I understand that RA and Dec are only related to time, and have nothing to do with the observation position on the earth. But when I use some star map apps to find the Moon, choose a datetime, the RA and Dec changes as I change GPS coordinates. Later I tried the "ICRF.radec()" func of skyfield, calc RA and Dec using "earth.at(t).observe(moon).radec()" and "(earth + wgs84.latlon(x * N, y * E)).at(t).observe(moon).radec()", and they gave me different results.
Is my understanding of RA and Dec wrong? Or is my usage of skyfield wrong?
Looking forward to your answers, thanks!

**Comment by brandon-rhodes** [comment]
Your understanding of RA and dec is, alas, wrong.

RA and dec tell you where an object is in the sky. For very distant stars, and for other galaxies, RA and dec will be essentially the same for everyone on Earth. Whether you are on one side of the globe or the other, the star Deneb is going to appear in the same direction, because the distance between two points on the globe is so terribly small compared to the distance to Deneb.

But think about the International Space Station. Does it appear in the same place in the sky for everyone on Earth? No. It is very close to the Earth, just about 250 miles above it, so people in one city might see the ISS right above them while folks in another city cannot see it at all. It will have a completely different RA and dec for people in two different cities.

The Moon is an intermediate case. It's far enough away that everyone on Earth sees it in front of the same set of stars on an given evening. But it's close enough that the distance between, say, New York and San Francisco is enough of a difference to place the Moon at a different exact position in the sky — and thus give it a different RA and dec.

**Reply by DieForSpicy** [reply]
Thank you very much for your patient answer!
So the RA and Dec are not calculated with the center of the earth as the origin, right?

**Reply by brandon-rhodes** [reply]
Correct!
